SwiftLintFramework.podspec

```diff
@@ -10,6 +10,6 @@ Pod::Spec.new do |s|
   s.source_files        = 'Source/SwiftLintFramework/**/*.swift'
   s.swift_versions      = ['5.2', '5.3']
   s.pod_target_xcconfig = { 'APPLICATION_EXTENSION_API_ONLY' => 'YES' }
-  s.dependency            'SourceKittenFramework', '~> 0.30.1'
+  s.dependency            'SourceKittenFramework', '~> 0.31.0'
```

Comment:
idk how this whole thing works with cp, but do you not need to specify swift-argument-parser here? I see we weren't doing commandant before either so maybe we're getting these transitively?

Reply:
This is a podspec for `SourceKittenFramework`, which is the framework that powers SwiftLint. It doesn't include the CLI frontend, which is in the `swiftlint` target. So we don't need SwiftyTextTable or swift-argument-parser.